test/integration/testUtils.js

```diff
@@ -1,7 +1,35 @@
+const esClient = require('../../app/lib/esClient');
+
+const maxWaitTimeMs = 3 * 60 * 1000;
+const gracePeriodMs = 5000;
+const pollingIntervalMs = 3000;
+
 function removeWhitespace(text) {
   return text && text.trim().replace(/\s+/g, ' ');
 }
 
+function esServerReady() {
+  // Note: yellow is the highest state for a single cluster
+  return esClient.getHealth().then(health => health[0].status === 'yellow').catch(() => false);
+}
+
+function waitForEsToStart(done, startTime) {
+  esServerReady().then((res) => {
+    if (res || (new Date() - startTime) > maxWaitTimeMs) {
```

Comment:
It looks like if the server hasn't started `done` will be executed which leads to the test continuing to run as if the server had started on this line https://github.com/nhsuk/profiles/pull/219/files?diff=split#diff-b919a538a0d33a82eb85ca756257c53aR22